upgrade-ssh-1.x-to-2.x.sh

```diff
@@ -253,20 +251,15 @@ for uuid in $UUIDS; do
     log "[$CURRENT_UPDATE/$NR_UPDATES] Updating $uuid on $SSH_HOST."
     log_filename="$uuid.upgrade2x.log"
 
-    if ! ssh "$SSH_HOST" -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o "Hostname=${uuid}.vpn" exit > /dev/null 2>&1; then
+    if ! ssh "$SSH_HOST" host -s "${uuid}" "exit" > /dev/null 2>&1; then
```

Comment:
What is the `host`?

Reply:
@agherzan this is the new connection method, have to connec to our backend, and send `host UUID` to connect over to that device. See https://github.com/resin-io/process/blob/master/process/support/accessing_user_accounts.md#feature-documentation and https://github.com/resin-io/resin-proxy/tree/e4e008ed6861d7c93924eceb218a595bccc5c9a3/src/services/sshproxy-scripts

Reply:
Got it.